ssg/build_cpe.py

```diff
@@ -72,7 +71,7 @@ def load_content_cpes(self):
                 continue
 
             # Get past "cpes" key, which was added for readability of the content
-            cpes_list = open_raw(dir_item_path)["cpes"]
+            cpes_list = open_and_macro_expand(dir_item_path, self.product_yaml)["cpes"]
```

Comment:
Only a small thing: I have noticed that all the compiled platforms in `build/rhel8/platforms/` have the `definition_location` key set to an empty string. Is it OK or do you think we should fill it in? It seems that removing it won't be easy because it's defined in the `XCCDFEntity` class which is a parent class of many other classes in build_yaml.py.